templates/about/release-cycle.html

```diff
@@ -653,7 +653,7 @@ <h2 id="charmed-kubernetes-release-cycle">Charmed Kubernetes<sup>&reg;</sup> rel
       </div>
     </div>
     <div class="row">
-      <p>For more information on previous and current releases of Charmed Kubernetes, please see the <a href="/kubernetes/docs/release-notes">Charmed Kubernetes release notes</a>.</p>
+      <p>For more information on previous and current releases, please see the <a href="https://ubuntu.com/kubernetes/docs/release-notes">Charmed Kubernetes</a> release notes or the <a href="https://microk8s.io/docs/release-notes">MicroK8s release notes.</a></p>
```

Comment:
Same here, can you change it to relative path `/kubernetes/docs/release-notes`?
And the microk8s link is an external link so please add the `class="p-link--external"`

Reply:
Done!